sources/editor/Xenko.Assets.Presentation/Templates/Core/ProjectExecutable.Windows/$ProjectName$.csproj.t4

```diff
@@ -2,11 +2,17 @@
 <Project Sdk="Microsoft.NET.Sdk">
 
   <PropertyGroup>
-    <TargetFramework>net461</TargetFramework>
+    <!-- We should switch Windows Game to netCore as well. Would help us find xplatform issues quicker -->
+    <TargetFramework>netcoreapp2.1</TargetFramework>
+    <RuntimeIdentifier>win-x64</RuntimeIdentifier>
+
     <ApplicationIcon>Resources\Icon.ico</ApplicationIcon>
     <OutputType>WinExe</OutputType>
     <RootNamespace><#= Properties.Namespace #></RootNamespace>
 
+    <!-- This resolves native dep publishing -->
+    <CopyLocalLockFileAssemblies>true</CopyLocalLockFileAssemblies>
```

Comment:
This will copy all assemblies to output folder (in non-publish mode).
Windows is working without this (dotnet resolves assemblies from NuGet and native dependencies are loaded from within NuGet).
With your other `CopyData` fix, is it really necessary?